aries-site/src/examples/components/menu/MenuHeaderExample.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Header, Menu } from 'grommet';
+import { AppIdentity } from 'aries-core';
+
+export const MenuHeaderExample = () => {
+  const items = [
+    { label: 'Change username' },
+    { label: 'Reset Password' },
+    { label: 'Logout' },
+  ];
+
+  return (
+    <Header background="background-contrast" pad="small" fill>
+      <AppIdentity brand="hpe" title="Service Name" />
```

Comment:
In the spirit of copy-paste, wondering if we should use `<AppIdentity>` directly here? Perhaps define it in this same file?

Perhaps we save this for a concentrated review of copy-pastability of examples?

Reply:
Very true, I think it's a question of if we would be publishing AppIdentity as a component in the design-system library. For now, I will define it in the file since the design system library is not published.

Reply:
Put it inline for now, leaning towards ability for people to copy and paste for now.